Stack/Opc.Ua.Core/Security/Certificates/DirectoryCertificateStore.cs

```diff
@@ -689,20 +697,28 @@ public async Task<X509CRLCollection> EnumerateCRLs(X509Certificate2 issuer, bool
             var crls = new X509CRLCollection();
             foreach (X509CRL crl in await EnumerateCRLs().ConfigureAwait(false))
```

Comment:
again, if enumerateCRL ignores the invalid/unknown CRL, there is no need to try/catch here?